frameworks/projects/MXRoyale/src/main/royale/mx/events/HTTPStatusEvent.as

```diff
@@ -19,91 +19,95 @@
+										cancelable:Boolean = false, status:int = 0)
+		{
+			super(type, bubbles, cancelable,status);
+		}
+		
 	}
-}
-
-/**
- *  Represents event objects that are dispatched when focus changes.
- *
- *  @see mx.core.UIComponent
- *  
- *  @langversion 3.0
- *  @playerversion Flash 9
- *  @playerversion AIR 1.1
- *  @productversion Royale 0.9.3
- *  @royalesuppresspublicvarwarning
- */
-COMPILE::JS
-public class HTTPStatusEvent extends org.apache.royale.events.Event
-{
-   /*  include "../core/Version.as"; */
-	public static const HTTP_STATUS:String = "httpStatus";
-	
-	//--------------------------------------------------------------------------
-	//
-	//  Class constants
-	//
-	//--------------------------------------------------------------------------
-
 	
-	//--------------------------------------------------------------------------
-	//
-	//  Constructor
-	//
-	//--------------------------------------------------------------------------
-
 	/**
-	 *  Constructor.
-	 *
-	 *  @param type The event type; indicates the action that caused the event.
-	 *
-	 *  @param bubbles Specifies whether the event can bubble
-	 *  up the display list hierarchy.
+	 *  Represents event objects that are dispatched when focus changes.
 	 *
-	 *  @param cancelable Specifies whether the behavior
-	 *  associated with the event can be prevented.
+	 *  @see mx.core.UIComponent
 	 *  
 	 *  @langversion 3.0
 	 *  @playerversion Flash 9
 	 *  @playerversion AIR 1.1
 	 *  @productversion Royale 0.9.3
+	 *  @royalesuppresspublicvarwarning
 	 */
-	public function HTTPStatusEvent(type:String, bubbles:Boolean = false,
-	cancelable:Boolean = false, status:int = 0)
+	COMPILE::JS
+	public class HTTPStatusEvent extends org.apache.royale.events.Event
 	{
-		super(type, bubbles, cancelable);
+		/*  include "../core/Version.as"; */
+		public static const HTTP_STATUS:String = "httpStatus";
+		private var m_status:int;
+		
+		//--------------------------------------------------------------------------
+		//
+		//  Class constants
+		//
+		//--------------------------------------------------------------------------
+		
+		
+		//--------------------------------------------------------------------------
+		//
+		//  Constructor
+		//
+		//--------------------------------------------------------------------------
+		
+		/**
+		 *  Constructor.
+		 *
+		 *  @param type The event type; indicates the action that caused the event.
+		 *
+		 *  @param bubbles Specifies whether the event can bubble
+		 *  up the display list hierarchy.
+		 *
+		 *  @param cancelable Specifies whether the behavior
+		 *  associated with the event can be prevented.
+		 *  
+		 *  @langversion 3.0
+		 *  @playerversion Flash 9
+		 *  @playerversion AIR 1.1
+		 *  @productversion Royale 0.9.3
+		 */
+		public function HTTPStatusEvent(type:String, bubbles:Boolean = false,
+										cancelable:Boolean = false, status:int = 0)
+		{
+			super(type, bubbles, cancelable);
+			this.m_status = status;

```

Comment:
I don't think you need 'this.'

There should be very few occurrences of 'this' in the AS source.  I think should only be needed when referencing member APIs that have the same name as local and parameter variables.

Reply:
Completely agree. They are irrelevant. Removed.